src/azure-cli/azure/cli/command_modules/eventgrid/tests/latest/test_eventgrid_commands.py

```diff
@@ -461,6 +461,7 @@ def test_create_topic(self, resource_group):
 
         self.cmd('az eventgrid topic delete --name {topic_name} --resource-group {rg}')
 
+    @unittest.skip('live test always fails, need fix by owners')
```

Comment:
Does this test fail in weekly live run?
If yes, we'd better replace "api version" in recording file and create an issue to owner to fix this test.

Reply:
Yes, it failed in weekly live run. It will fail even with recording mode. So we have to skip it.